compiler/kernel/op.go

```diff
@@ -325,7 +416,21 @@ func (b *Builder) compileSequential(seq *dag.Sequential, parents []zbuf.Puller)
 	return parents, nil
 }
 
-func (b *Builder) compileParallel(par *dag.Parallel, parents []zbuf.Puller) ([]zbuf.Puller, error) {
+func (b *Builder) compileParallel(parallel *dag.Parallel, parents []zbuf.Puller) ([]zbuf.Puller, error) {
+	if parallel.Any {
+		if len(parents) != 1 {
+			return nil, errors.New("internal compiler error: any-path parallel dag operators requires a single parent")
+		}
+		var ops []zbuf.Puller
+		for _, o := range parallel.Ops {
+			op, err := b.compile(o, []zbuf.Puller{parents[0]})
```

Comment:
Nit: A slice expression would be simpler.
```suggestion
			op, err := b.compile(o, parents[:1])
```